src/server/snapshot.cc

```diff
@@ -272,24 +272,36 @@ void SliceSnapshot::OnDbChange(DbIndex db_index, const DbSlice::ChangeReq& req)
   }
 }
 
+// For any key any journal entry must arrive at the replica strictly after its first original rdb
+// value.
+// TODO: We should skip it if on db change was called - possible bug!
+// For example: l =  {v1, v2}, LPUSH l v3 writes {v1, v2, v3} in rdb, and sends LPUSH l v3
```

Comment:
But the on db change should be called before the change.
So in this example the on db change will wirte v1,v2 to the rdb

Reply:
Ah, yes, it will have a higher snapshot version 😅 So it should be fine in all cases, I guess